src/common/components/Box/Box.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
+
+import './Box.scss';
+
+class Box extends Component {
+  render() {
+    const { title, actions, loading } = this.props;
+
+    return (
+      <div className="__Box__">
+        <Card title={title} loading={loading}>
+          <div className="pa3">{this.props.children}</div>
+          <div className="pb3">{actions}</div>
+        </Card>
+      </div>
+    );
+  }
+}
+
+Box.propTypes = {
+  actions: PropTypes.node,
```

Comment:
actions sounds like it'd be `[<button>action1</action>, <button>action2...]`, maybe we can make it like that to be consistent with `antd`. Although it would work exactly same -_-